### DISCUSSION TITLE
Qubino ZMNHND1 I2 entry

### DISCUSSION THREAD

**Discussion by quatsch** [zwave-js/zwave-js#5945] [Q&A] [answered]
Hello,

I'm using a Qubino ZMNHND1 with ZwaveJS and it works fine
I would like to be able to use the i2 entry.
Therefor I activated parameter 100 to 9 (binary Sensor)
Once I do this, i2 still doesn't come back and for some reason when I re-interview the node at this stage it totally fails. The only way to get it back running is to set parameter 100 back to 0 (disabled)
Is there anything else I could try to get this input functional under ZwaveJS ?

Br, Quatsch

**Comment by kpine** [comment, marked as answer]
Did you see the note in the manual for that parameter?

> NOTE 1: After changing the values of the parameter, first exclude the device (without setting the parameters to their default values), then wait at least 30 seconds to re-include the device!

Did you exclude and re-include? Re-interview is not equivalent.

**Reply by quatsch** [reply]
Okay I'm feeling so stupid, spent the whole evening on this.
But this works great :) and the sensor pop's up in Home Assistant automatically !
